src/data/vault-install.json

```diff
@@ -1,34 +1,22 @@
 {
   "featuredTutorials": [
     {
-      "title": "Getting Started",
-      "description": "A quick tour of common Vault features to jump-start.",
-      "href": "https://learn.hashicorp.com/collections/vault/getting-started"
+      "collectionSlug": "vault/getting-started"
```

Comment:
I've used an array of objects here so that:
1. A mix of collections and tutorials can be used (to match previous data)
2. So that the data works nicely with existing "extract inline content" utilities, which look for `collectionSlug` and `tutorialSlug` keys

Reply:
This makes sense to me! 👍🏻 Making the property names match the expected key names is helpful too.